frontend-opal/src/components/SearchResult/SearchResult.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import styles from './styles/index';
+
+import ResultLists from '../../container/SearchResult/ResultLists';
+import SearchHeader from '../../container/SearchResult/SearchHeader';
+
+function Index() {
+  return (
+    <View style={styles.Index}>
+      <SearchHeader />
+      <ResultLists />
```

Comment:
@Yuan-EPAM technically this is going to be only one `ResultList` or multiple `Results`